plugins/auth-backend/src/identity/IdentityClient.ts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import fetch from 'cross-fetch';
+import { JWK, JWT, JWKS, JSONWebKey } from 'jose';
```

Comment:
Shiet, realized #4047 ported `auth-backend` to jose v3, so will need to update usage here as well >_<

I could take a stab at that @erikxiv if you want

Reply:
Be my guest! I am after the feature, not my specific code ;)

Could you perchance fix the error too, my package.json-addition to create-app complains about a package version not being available 🙄

Reply:
Ended up having to revert back to v1 anyway, https://github.com/backstage/backstage/pull/4184, will see if we can get this merged for today's release